documentation/modules/proc-installing-product.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// assembly-evaluation.adoc
+
+[id='proc-install-product-{context}']
+= Installing {ProductName}
+
+Using the xref:proc-product-downloads-{context}[download files], install {ProductName} with the Custom Resource Definitions (CRDs) and RBAC configuration required for deployment.
+
+.Prerequisites
+
+* Installation requires a Kubernetes account with cluster admin credentials.
+
+.Procedure
+
+. Log in to the Kubernetes cluster with cluster admin privileges.
+
+. Modify the installation files to reference a new `kafka` namespace where you will install the {ProductName} Kafka Cluster Operator.
++
+NOTE: By default, the files work in the `myproject` namespace.
++
+* On Linux, use:
++
+
+[source, shell, subs=+quotes]
+----
+sed -i 's/namespace: .\*/namespace: kafka/' install/cluster-operator/*RoleBinding*.yaml
+----
++
+* On Mac, use:
++
+[source, shell, subs=+quotes]
+----
+sed -i '' 's/namespace: .\*/namespace: kafka/' install/cluster-operator/*RoleBinding*.yaml
+----
+
+. Create the new `kafka` namespace for the {ProductName} Kafka Cluster Operator.
```

Comment:
It still seems backwards to me that we create the namespace only after having told the read about this kafka namespace and had then change the resources. Why can't the namespace creation be the first step?

Reply:
I guess the idea was to do the prep first. But I've switched.